--- a/Makefile.real
+++ b/Makefile.real
@@ -89,8 +89,8 @@
 
 .PHONY: install
 install: modules
-	@$(MAKE) -C $(KLIB_BUILD) M=$(BACKPORT_DIR)			\
-		INSTALL_MOD_DIR=$(KMODDIR) $(KMODPATH_ARG)		\
+	@$(MAKE) -C $(KLIB_BUILD) M=$(BACKPORT_DIR) \
+		INSTALL_MOD_PATH=$(shell pwd)/debian/tmp/$(KLIB)/$(KMODDIR) $(KMODPATH_ARG) \
 		modules_install
 	@./scripts/blacklist.sh $(KLIB)/ $(KLIB)/$(KMODDIR)
 	@./scripts/compress_modules.sh $(KLIB)/$(KMODDIR)
